### ISSUE TITLE
k value in the kl distance equation of 3D data in 3d_autoencoderkl_tutorial.py

### ISSUE THREAD

**Issue by minghuix98** [Project-MONAI/GenerativeModels#501]
If the dimension k=3 for three dimensional data, then the following -1 should be -3/2?
`line 259: kl_loss = 0.5 * torch.sum(z_mu.pow(2) + z_sigma.pow(2) - torch.log(z_sigma.pow(2)) - 1, dim=[1, 2, 3, 4])`

**Comment by virginiafdez** [comment]
Hello!

Could you provide more details about what you mean with -3/2?

Thanks.

**Comment by minghuix98** [comment]
Thanks for your reply! 
Sorry, it’s probably my misunderstanding. Now I think it's correct.
